### ISSUE TITLE
idle_threshold setting doesn't work on Wayland

### ISSUE THREAD

**Issue by dismint** [dunst-project/dunst#1214]
### Issue description
dunst is unable to process the `idle_threshold` setting on Wayland. It simply returns a warning in the log that it is unable to set the variable. When using `force_wayland=true`, dunst reports it cannot process `idle_threshold`, which is to be expected considering the man page reads:
```
Don't timeout notifications if user is idle longer than this time. See TIME FORMAT for valid times.
Set to 0 to disable.

A client can mark a notification as transient to bypass this setting and timeout anyway. Use a rule with 'set_transient = no' to disable this behavior.

Note: this doesn't work on xwayland.
```
When run on X11, it works as expected.

Running on Wayland produces the following results from `dunst -verbosity debug`:

```
INFO: Using Wayland output
DEBUG: [   handle_global:0428] Found toplevel manager 28
INFO: New output found - id 0
DEBUG: [   handle_global:0422] Binding to output 50
DEBUG: Using cross-namespace EXTERNAL authentication (this will deadlock if server is GDBus < 2.73.3)
DEBUG: [         wake_up:0063] Ignoring wake up
WARNING: couldn't set idle timeout
DEBUG: [             run:0077] RUN, reason 0
INFO: Idle status queried: 0
INFO: Wayland: Hiding window
```
### Installation info

- Version: `1.9.2`
- Install type: arch `extra/dunst`
- Window manager / Desktop environment: `Hyprland`
- Distro: `EndeavourOS`

<details>
<summary>dunstrc</summary>
```ini
[global]
	# overall styling
	monitor = 0
	follow = mouse
	enable_posix_regex = true
	notification_limit = 2

	# styling of each notification
	width = 300
	origin = top-center
	offset = 0x25
	frame_width = 4
	corner_radius = 4

	gap_size = 10

	idle_threshold = 120ms

	sticky_history = true

	mouse_left_click = do_action, close_current
```
</details>

**Comment by Narrat** [comment]
Did you try the setting without the SI unit?

**Comment by dismint** [comment]
Yes I've tried it with `ms`, without `ms`, in quotes with `ms`, in quotes without `ms`. All lead to this same result.

**Comment by Narrat** [comment]
Okay, because I'm not sure if dunst capable of doing this unit conversion.
But indeed. I can see it in the debug output too. Was previously using the default and hadn't set the config option. But the default seems to be broken too, as notifications timed out which shouldn't have. And i know it was working in the past :D
If I get around to I will try to bisect this.

**Comment by dismint** [comment]
I went back and downgraded the dunst package, this issue seems to persist even through 1.6.0 when Wayland was first supported. If it's like you said and this worked at some point in time but broke recently, I'm leaning towards a dependency failing since it doesn't seem to have worked at any point in time by solely downgrading dunst.

**Comment by Narrat** [comment]
Dependency or (wayland-)protocol . At least it is enlightening that a simple downgrade doesn't seem to be working.
Memories can be tricky and unreliable, but I'm farily sure I'm not misremembering things :D  When the wayland implementation was WIP it missed timeout in general at first. Every notification needed to be dismissed.
Then idle got added and at that time I was using hikari and tested the stuff. And I remember that after time x even the non urgent notifications weren't disappearing. Easily visible due to the added timer text.
But that was years ago... it should probably taken with a grain of salt :D And as I cannot tell when it stopped working. I'm not using notifications that much and it was apparent I missed this possible change of behaviour.

We will see. I probably should read the issues and merge requests from back then again.

**Comment by fwsmit** [comment]
Dunst is using the Kde idle protocol, which was the only available option at the time and pretty widely supported (not only kde). When looking at [this commit](https://github.com/swaywm/sway/commit/7cf4e1d5c68d764037f78cbffb167a4cc763c6c1) from sway, it seems like there is a new standard protocol that has been introduced. Dunst doesn't have support for that yet.

**Comment by Narrat** [comment]
Got finally around for some testing (stupid cold...) and the tl;dr is: Didn't seem to work. Neither setting the value, nor using the default.

Hikari (built with wlroots 0.15.x)
```
interface: 'wl_shm',                                     version:  1, name:  1
	formats (fourcc):
	0x48344241 = 'AB4H'
	0x48344258 = 'XB4H'
	0x30334241 = 'AB30'
	0x30334258 = 'XB30'
	0x36314752 = 'RG16'
	0x35314152 = 'RA15'
	0x35315852 = 'RX15'
	0x32314152 = 'RA12'
	0x32315852 = 'RX12'
	0x34324742 = 'BG24'
	0x34324241 = 'AB24'
	0x34324258 = 'XB24'
	         1 = 'XR24'
	         0 = 'AR24'
interface: 'wl_drm',                                     version:  2, name:  2
interface: 'zwp_linux_dmabuf_v1',                        version:  4, name:  3
	main device: 0xE280 (/dev/dri/card1 or /dev/dri/renderD128)
	tranche
		target device: 0xE280 (/dev/dri/card1 or /dev/dri/renderD128)
		flags: none
		formats (fourcc) and modifiers (names):
		0x48344241 = 'AB4H'; 0x00ffffffffffffff = INVALID
		0x48344241 = 'AB4H'; 0x00ffffffffffffff = INVALID
		0x48344241 = 'AB4H'; 0x0000000000000000 = LINEAR
		0x48344258 = 'XB4H'; 0x00ffffffffffffff = INVALID
		0x48344258 = 'XB4H'; 0x00ffffffffffffff = INVALID
		0x48344258 = 'XB4H'; 0x0000000000000000 = LINEAR
		0x38344241 = 'AB48'; 0x00ffffffffffffff = INVALID
		0x38344241 = 'AB48'; 0x00ffffffffffffff = INVALID
		0x38344241 = 'AB48'; 0x0000000000000000 = LINEAR
		0x38344258 = 'XB48'; 0x00ffffffffffffff = INVALID
		0x38344258 = 'XB48'; 0x00ffffffffffffff = INVALID
		0x38344258 = 'XB48'; 0x0000000000000000 = LINEAR
		0x30335241 = 'AR30'; 0x00ffffffffffffff = INVALID
		0x30335241 = 'AR30'; 0x00ffffffffffffff = INVALID
		0x30335241 = 'AR30'; 0x0000000000000000 = LINEAR
		0x30335258 = 'XR30'; 0x00ffffffffffffff = INVALID
		0x30335258 = 'XR30'; 0x00ffffffffffffff = INVALID
		0x30335258 = 'XR30'; 0x0000000000000000 = LINEAR
		0x30334241 = 'AB30'; 0x00ffffffffffffff = INVALID
		0x30334241 = 'AB30'; 0x00ffffffffffffff = INVALID
		0x30334241 = 'AB30'; 0x0000000000000000 = LINEAR
		0x30334258 = 'XB30'; 0x00ffffffffffffff = INVALID
		0x30334258 = 'XB30'; 0x00ffffffffffffff = INVALID
		0x30334258 = 'XB30'; 0x0000000000000000 = LINEAR
		0x34325241 = 'AR24'; 0x00ffffffffffffff = INVALID
		0x34325241 = 'AR24'; 0x00ffffffffffffff = INVALID
		0x34325241 = 'AR24'; 0x0000000000000000 = LINEAR
		0x34324241 = 'AB24'; 0x00ffffffffffffff = INVALID
		0x34324241 = 'AB24'; 0x00ffffffffffffff = INVALID
		0x34324241 = 'AB24'; 0x0000000000000000 = LINEAR
		0x34325258 = 'XR24'; 0x00ffffffffffffff = INVALID
		0x34325258 = 'XR24'; 0x00ffffffffffffff = INVALID
		0x34325258 = 'XR24'; 0x0000000000000000 = LINEAR
		0x34324258 = 'XB24'; 0x00ffffffffffffff = INVALID
		0x34324258 = 'XB24'; 0x00ffffffffffffff = INVALID
		0x34324258 = 'XB24'; 0x0000000000000000 = LINEAR
		0x35315241 = 'AR15'; 0x00ffffffffffffff = INVALID
		0x35315241 = 'AR15'; 0x00ffffffffffffff = INVALID
		0x35315241 = 'AR15'; 0x0000000000000000 = LINEAR
		0x36314752 = 'RG16'; 0x00ffffffffffffff = INVALID
		0x36314752 = 'RG16'; 0x00ffffffffffffff = INVALID
		0x36314752 = 'RG16'; 0x0000000000000000 = LINEAR
		0x20203852 = 'R8  '; 0x00ffffffffffffff = INVALID
		0x20203852 = 'R8  '; 0x00ffffffffffffff = INVALID
		0x20203852 = 'R8  '; 0x0000000000000000 = LINEAR
		0x20363152 = 'R16 '; 0x00ffffffffffffff = INVALID
		0x20363152 = 'R16 '; 0x00ffffffffffffff = INVALID
		0x20363152 = 'R16 '; 0x0000000000000000 = LINEAR
		0x38385247 = 'GR88'; 0x00ffffffffffffff = INVALID
		0x38385247 = 'GR88'; 0x00ffffffffffffff = INVALID
		0x38385247 = 'GR88'; 0x0000000000000000 = LINEAR
		0x32335247 = 'GR32'; 0x00ffffffffffffff = INVALID
		0x32335247 = 'GR32'; 0x00ffffffffffffff = INVALID
		0x32335247 = 'GR32'; 0x0000000000000000 = LINEAR
		0x39565559 = 'YUV9'; 0x00ffffffffffffff = INVALID
		0x39565559 = 'YUV9'; 0x00ffffffffffffff = INVALID
		0x39565559 = 'YUV9'; 0x0000000000000000 = LINEAR
		0x31315559 = 'YU11'; 0x00ffffffffffffff = INVALID
		0x31315559 = 'YU11'; 0x00ffffffffffffff = INVALID
		0x31315559 = 'YU11'; 0x0000000000000000 = LINEAR
		0x32315559 = 'YU12'; 0x00ffffffffffffff = INVALID
		0x32315559 = 'YU12'; 0x00ffffffffffffff = INVALID
		0x32315559 = 'YU12'; 0x0000000000000000 = LINEAR
		0x36315559 = 'YU16'; 0x00ffffffffffffff = INVALID
		0x36315559 = 'YU16'; 0x00ffffffffffffff = INVALID
		0x36315559 = 'YU16'; 0x0000000000000000 = LINEAR
		0x34325559 = 'YU24'; 0x00ffffffffffffff = INVALID
		0x34325559 = 'YU24'; 0x00ffffffffffffff = INVALID
		0x34325559 = 'YU24'; 0x0000000000000000 = LINEAR
		0x39555659 = 'YVU9'; 0x00ffffffffffffff = INVALID
		0x39555659 = 'YVU9'; 0x00ffffffffffffff = INVALID
		0x39555659 = 'YVU9'; 0x0000000000000000 = LINEAR
		0x31315659 = 'YV11'; 0x00ffffffffffffff = INVALID
		0x31315659 = 'YV11'; 0x00ffffffffffffff = INVALID
		0x31315659 = 'YV11'; 0x0000000000000000 = LINEAR
		0x32315659 = 'YV12'; 0x00ffffffffffffff = INVALID
		0x32315659 = 'YV12'; 0x00ffffffffffffff = INVALID
		0x32315659 = 'YV12'; 0x0000000000000000 = LINEAR
		0x36315659 = 'YV16'; 0x00ffffffffffffff = INVALID
		0x36315659 = 'YV16'; 0x00ffffffffffffff = INVALID
		0x36315659 = 'YV16'; 0x0000000000000000 = LINEAR
		0x34325659 = 'YV24'; 0x00ffffffffffffff = INVALID
		0x34325659 = 'YV24'; 0x00ffffffffffffff = INVALID
		0x34325659 = 'YV24'; 0x0000000000000000 = LINEAR
		0x3231564e = 'NV12'; 0x00ffffffffffffff = INVALID
		0x3231564e = 'NV12'; 0x00ffffffffffffff = INVALID
		0x3231564e = 'NV12'; 0x0000000000000000 = LINEAR
		0x3132564e = 'NV21'; 0x00ffffffffffffff = INVALID
		0x3132564e = 'NV21'; 0x00ffffffffffffff = INVALID
		0x3132564e = 'NV21'; 0x0000000000000000 = LINEAR
		0x30313050 = 'P010'; 0x00ffffffffffffff = INVALID
		0x30313050 = 'P010'; 0x00ffffffffffffff = INVALID
		0x30313050 = 'P010'; 0x0000000000000000 = LINEAR
		0x32313050 = 'P012'; 0x00ffffffffffffff = INVALID
		0x32313050 = 'P012'; 0x00ffffffffffffff = INVALID
		0x32313050 = 'P012'; 0x0000000000000000 = LINEAR
		0x36313050 = 'P016'; 0x00ffffffffffffff = INVALID
		0x36313050 = 'P016'; 0x00ffffffffffffff = INVALID
		0x36313050 = 'P016'; 0x0000000000000000 = LINEAR
		0x30333050 = 'P030'; 0x00ffffffffffffff = INVALID
		0x30333050 = 'P030'; 0x00ffffffffffffff = INVALID
		0x30333050 = 'P030'; 0x0000000000000000 = LINEAR
		0x3631564e = 'NV16'; 0x00ffffffffffffff = INVALID
		0x3631564e = 'NV16'; 0x00ffffffffffffff = INVALID
		0x3631564e = 'NV16'; 0x0000000000000000 = LINEAR
		0x56555941 = 'AYUV'; 0x00ffffffffffffff = INVALID
		0x56555941 = 'AYUV'; 0x00ffffffffffffff = INVALID
		0x56555941 = 'AYUV'; 0x0000000000000000 = LINEAR
		0x56555958 = 'XYUV'; 0x00ffffffffffffff = INVALID
		0x56555958 = 'XYUV'; 0x00ffffffffffffff = INVALID
		0x56555958 = 'XYUV'; 0x0000000000000000 = LINEAR
		0x30313459 = 'Y410'; 0x00ffffffffffffff = INVALID
		0x30313459 = 'Y410'; 0x00ffffffffffffff = INVALID
		0x30313459 = 'Y410'; 0x0000000000000000 = LINEAR
		0x32313459 = 'Y412'; 0x00ffffffffffffff = INVALID
		0x32313459 = 'Y412'; 0x00ffffffffffffff = INVALID
		0x32313459 = 'Y412'; 0x0000000000000000 = LINEAR
		0x36313459 = 'Y416'; 0x00ffffffffffffff = INVALID
		0x36313459 = 'Y416'; 0x00ffffffffffffff = INVALID
		0x36313459 = 'Y416'; 0x0000000000000000 = LINEAR
		0x56595559 = 'YUYV'; 0x00ffffffffffffff = INVALID
		0x56595559 = 'YUYV'; 0x00ffffffffffffff = INVALID
		0x56595559 = 'YUYV'; 0x0000000000000000 = LINEAR
		0x55595659 = 'YVYU'; 0x00ffffffffffffff = INVALID
		0x55595659 = 'YVYU'; 0x00ffffffffffffff = INVALID
		0x55595659 = 'YVYU'; 0x0000000000000000 = LINEAR
		0x59565955 = 'UYVY'; 0x00ffffffffffffff = INVALID
		0x59565955 = 'UYVY'; 0x00ffffffffffffff = INVALID
		0x59565955 = 'UYVY'; 0x0000000000000000 = LINEAR
		0x59555956 = 'VYUY'; 0x00ffffffffffffff = INVALID
		0x59555956 = 'VYUY'; 0x00ffffffffffffff = INVALID
		0x59555956 = 'VYUY'; 0x0000000000000000 = LINEAR
		0x30313259 = 'Y210'; 0x00ffffffffffffff = INVALID
		0x30313259 = 'Y210'; 0x00ffffffffffffff = INVALID
		0x30313259 = 'Y210'; 0x0000000000000000 = LINEAR
		0x32313259 = 'Y212'; 0x00ffffffffffffff = INVALID
		0x32313259 = 'Y212'; 0x00ffffffffffffff = INVALID
		0x32313259 = 'Y212'; 0x0000000000000000 = LINEAR
		0x36313259 = 'Y216'; 0x00ffffffffffffff = INVALID
		0x36313259 = 'Y216'; 0x00ffffffffffffff = INVALID
		0x36313259 = 'Y216'; 0x0000000000000000 = LINEAR
interface: 'wl_compositor',                              version:  4, name:  4
interface: 'wl_subcompositor',                           version:  1, name:  5
interface: 'wl_data_device_manager',                     version:  3, name:  6
interface: 'zxdg_output_manager_v1',                     version:  3, name:  7
	xdg_output_v1
		output: 19
		name: 'DP-1'
		description: 'Acer Technologies GF246 T8CEE0018521 (DP-1)'
		logical_x: 0, logical_y: 0
		logical_width: 1920, logical_height: 1080
interface: 'zwlr_gamma_control_manager_v1',              version:  1, name:  8
interface: 'zwlr_screencopy_manager_v1',                 version:  3, name:  9
interface: 'zwp_virtual_keyboard_manager_v1',            version:  1, name: 10
interface: 'zwlr_virtual_pointer_manager_v1',            version:  2, name: 11
interface: 'org_kde_kwin_server_decoration_manager',     version:  1, name: 12
interface: 'zxdg_decoration_manager_v1',                 version:  1, name: 13
interface: 'zwlr_data_control_manager_v1',               version:  2, name: 14
interface: 'zwp_primary_selection_device_manager_v1',    version:  1, name: 15
interface: 'wl_seat',                                    version:  7, name: 16
	name: seat0
	capabilities: pointer keyboard
	keyboard repeat rate: 25
	keyboard repeat delay: 600
interface: 'xdg_wm_base',                                version:  2, name: 17
interface: 'zwlr_layer_shell_v1',                        version:  4, name: 18
interface: 'wl_output',                                  version:  4, name: 19
	name: DP-1
	description: Acer Technologies GF246 T8CEE0018521 (DP-1)
	x: 0, y: 0, scale: 1,
	physical_width: 530 mm, physical_height: 300 mm,
	make: 'Acer Technologies', model: 'GF246',
	subpixel_orientation: unknown, output_transform: normal,
	mode:
		width: 1920 px, height: 1080 px, refresh: 60.000 Hz,
		flags: current
```

Labwc (built with wlroots 0.16.x
```
interface: 'wl_shm',                                     version:  1, name:  1
	formats (fourcc):
	0x38344241 = 'AB48'
	0x38344258 = 'XB48'
	0x48344241 = 'AB4H'
	0x48344258 = 'XB4H'
	0x30334241 = 'AB30'
	0x30334258 = 'XB30'
	0x36314752 = 'RG16'
	0x35314152 = 'RA15'
	0x35315852 = 'RX15'
	0x32314152 = 'RA12'
	0x32315852 = 'RX12'
	0x34324742 = 'BG24'
	0x34324241 = 'AB24'
	0x34324258 = 'XB24'
	         1 = 'XR24'
	         0 = 'AR24'
interface: 'wl_drm',                                     version:  2, name:  2
interface: 'zwp_linux_dmabuf_v1',                        version:  4, name:  3
	main device: 0xE280 (/dev/dri/card1 or /dev/dri/renderD128)
	tranche
		target device: 0xE280 (/dev/dri/card1 or /dev/dri/renderD128)
		flags: none
		formats (fourcc) and modifiers (names):
		0x48344241 = 'AB4H'; 0x00ffffffffffffff = INVALID
		0x48344241 = 'AB4H'; 0x0000000000000000 = LINEAR
		0x48344258 = 'XB4H'; 0x00ffffffffffffff = INVALID
		0x48344258 = 'XB4H'; 0x0000000000000000 = LINEAR
		0x38344241 = 'AB48'; 0x00ffffffffffffff = INVALID
		0x38344241 = 'AB48'; 0x0000000000000000 = LINEAR
		0x38344258 = 'XB48'; 0x00ffffffffffffff = INVALID
		0x38344258 = 'XB48'; 0x0000000000000000 = LINEAR
		0x30335241 = 'AR30'; 0x00ffffffffffffff = INVALID
		0x30335241 = 'AR30'; 0x0000000000000000 = LINEAR
		0x30335258 = 'XR30'; 0x00ffffffffffffff = INVALID
		0x30335258 = 'XR30'; 0x0000000000000000 = LINEAR
		0x30334241 = 'AB30'; 0x00ffffffffffffff = INVALID
		0x30334241 = 'AB30'; 0x0000000000000000 = LINEAR
		0x30334258 = 'XB30'; 0x00ffffffffffffff = INVALID
		0x30334258 = 'XB30'; 0x0000000000000000 = LINEAR
		0x34325241 = 'AR24'; 0x00ffffffffffffff = INVALID
		0x34325241 = 'AR24'; 0x0000000000000000 = LINEAR
		0x34324241 = 'AB24'; 0x00ffffffffffffff = INVALID
		0x34324241 = 'AB24'; 0x0000000000000000 = LINEAR
		0x34325258 = 'XR24'; 0x00ffffffffffffff = INVALID
		0x34325258 = 'XR24'; 0x0000000000000000 = LINEAR
		0x34324258 = 'XB24'; 0x00ffffffffffffff = INVALID
		0x34324258 = 'XB24'; 0x0000000000000000 = LINEAR
		0x35315241 = 'AR15'; 0x00ffffffffffffff = INVALID
		0x35315241 = 'AR15'; 0x0000000000000000 = LINEAR
		0x36314752 = 'RG16'; 0x00ffffffffffffff = INVALID
		0x36314752 = 'RG16'; 0x0000000000000000 = LINEAR
		0x20203852 = 'R8  '; 0x00ffffffffffffff = INVALID
		0x20203852 = 'R8  '; 0x0000000000000000 = LINEAR
		0x20363152 = 'R16 '; 0x00ffffffffffffff = INVALID
		0x20363152 = 'R16 '; 0x0000000000000000 = LINEAR
		0x38385247 = 'GR88'; 0x00ffffffffffffff = INVALID
		0x38385247 = 'GR88'; 0x0000000000000000 = LINEAR
		0x32335247 = 'GR32'; 0x00ffffffffffffff = INVALID
		0x32335247 = 'GR32'; 0x0000000000000000 = LINEAR
		0x39565559 = 'YUV9'; 0x00ffffffffffffff = INVALID
		0x39565559 = 'YUV9'; 0x0000000000000000 = LINEAR
		0x31315559 = 'YU11'; 0x00ffffffffffffff = INVALID
		0x31315559 = 'YU11'; 0x0000000000000000 = LINEAR
		0x32315559 = 'YU12'; 0x00ffffffffffffff = INVALID
		0x32315559 = 'YU12'; 0x0000000000000000 = LINEAR
		0x36315559 = 'YU16'; 0x00ffffffffffffff = INVALID
		0x36315559 = 'YU16'; 0x0000000000000000 = LINEAR
		0x34325559 = 'YU24'; 0x00ffffffffffffff = INVALID
		0x34325559 = 'YU24'; 0x0000000000000000 = LINEAR
		0x39555659 = 'YVU9'; 0x00ffffffffffffff = INVALID
		0x39555659 = 'YVU9'; 0x0000000000000000 = LINEAR
		0x31315659 = 'YV11'; 0x00ffffffffffffff = INVALID
		0x31315659 = 'YV11'; 0x0000000000000000 = LINEAR
		0x32315659 = 'YV12'; 0x00ffffffffffffff = INVALID
		0x32315659 = 'YV12'; 0x0000000000000000 = LINEAR
		0x36315659 = 'YV16'; 0x00ffffffffffffff = INVALID
		0x36315659 = 'YV16'; 0x0000000000000000 = LINEAR
		0x34325659 = 'YV24'; 0x00ffffffffffffff = INVALID
		0x34325659 = 'YV24'; 0x0000000000000000 = LINEAR
		0x3231564e = 'NV12'; 0x00ffffffffffffff = INVALID
		0x3231564e = 'NV12'; 0x0000000000000000 = LINEAR
		0x3132564e = 'NV21'; 0x00ffffffffffffff = INVALID
		0x3132564e = 'NV21'; 0x0000000000000000 = LINEAR
		0x30313050 = 'P010'; 0x00ffffffffffffff = INVALID
		0x30313050 = 'P010'; 0x0000000000000000 = LINEAR
		0x32313050 = 'P012'; 0x00ffffffffffffff = INVALID
		0x32313050 = 'P012'; 0x0000000000000000 = LINEAR
		0x36313050 = 'P016'; 0x00ffffffffffffff = INVALID
		0x36313050 = 'P016'; 0x0000000000000000 = LINEAR
		0x30333050 = 'P030'; 0x00ffffffffffffff = INVALID
		0x30333050 = 'P030'; 0x0000000000000000 = LINEAR
		0x3631564e = 'NV16'; 0x00ffffffffffffff = INVALID
		0x3631564e = 'NV16'; 0x0000000000000000 = LINEAR
		0x56555941 = 'AYUV'; 0x00ffffffffffffff = INVALID
		0x56555941 = 'AYUV'; 0x0000000000000000 = LINEAR
		0x56555958 = 'XYUV'; 0x00ffffffffffffff = INVALID
		0x56555958 = 'XYUV'; 0x0000000000000000 = LINEAR
		0x30313459 = 'Y410'; 0x00ffffffffffffff = INVALID
		0x30313459 = 'Y410'; 0x0000000000000000 = LINEAR
		0x32313459 = 'Y412'; 0x00ffffffffffffff = INVALID
		0x32313459 = 'Y412'; 0x0000000000000000 = LINEAR
		0x36313459 = 'Y416'; 0x00ffffffffffffff = INVALID
		0x36313459 = 'Y416'; 0x0000000000000000 = LINEAR
		0x56595559 = 'YUYV'; 0x00ffffffffffffff = INVALID
		0x56595559 = 'YUYV'; 0x0000000000000000 = LINEAR
		0x55595659 = 'YVYU'; 0x00ffffffffffffff = INVALID
		0x55595659 = 'YVYU'; 0x0000000000000000 = LINEAR
		0x59565955 = 'UYVY'; 0x00ffffffffffffff = INVALID
		0x59565955 = 'UYVY'; 0x0000000000000000 = LINEAR
		0x59555956 = 'VYUY'; 0x00ffffffffffffff = INVALID
		0x59555956 = 'VYUY'; 0x0000000000000000 = LINEAR
		0x30313259 = 'Y210'; 0x00ffffffffffffff = INVALID
		0x30313259 = 'Y210'; 0x0000000000000000 = LINEAR
		0x32313259 = 'Y212'; 0x00ffffffffffffff = INVALID
		0x32313259 = 'Y212'; 0x0000000000000000 = LINEAR
		0x36313259 = 'Y216'; 0x00ffffffffffffff = INVALID
		0x36313259 = 'Y216'; 0x0000000000000000 = LINEAR
interface: 'zxdg_output_manager_v1',                     version:  3, name:  4
	xdg_output_v1
		output: 36
		name: 'DP-1'
		description: 'Acer Technologies GF246 T8CEE0018521 (DP-1)'
		logical_x: 0, logical_y: 0
		logical_width: 1920, logical_height: 1080
interface: 'zwlr_output_manager_v1',                     version:  4, name:  5
interface: 'wl_compositor',                              version:  5, name:  6
interface: 'wl_subcompositor',                           version:  1, name:  7
interface: 'wl_data_device_manager',                     version:  3, name:  8
interface: 'zwp_primary_selection_device_manager_v1',    version:  1, name:  9
interface: 'wl_seat',                                    version:  8, name: 10
	name: seat0
	capabilities: pointer keyboard
	keyboard repeat rate: 25
	keyboard repeat delay: 600
interface: 'zwlr_virtual_pointer_manager_v1',            version:  2, name: 11
interface: 'zwp_virtual_keyboard_manager_v1',            version:  1, name: 12
interface: 'zwp_pointer_gestures_v1',                    version:  3, name: 13
interface: 'xdg_wm_base',                                version:  2, name: 14
interface: 'org_kde_kwin_server_decoration_manager',     version:  1, name: 15
interface: 'zxdg_decoration_manager_v1',                 version:  1, name: 16
interface: 'xdg_activation_v1',                          version:  1, name: 17
interface: 'wp_presentation',                            version:  1, name: 18
	presentation clock id: 1 (CLOCK_MONOTONIC)
interface: 'zwlr_export_dmabuf_manager_v1',              version:  1, name: 19
interface: 'zwlr_screencopy_manager_v1',                 version:  3, name: 20
interface: 'zwlr_data_control_manager_v1',               version:  2, name: 21
interface: 'zwlr_gamma_control_manager_v1',              version:  1, name: 22
interface: 'wp_viewporter',                              version:  1, name: 23
interface: 'wp_single_pixel_buffer_manager_v1',          version:  1, name: 24
interface: 'org_kde_kwin_idle',                          version:  1, name: 25
interface: 'ext_idle_notifier_v1',                       version:  1, name: 26
interface: 'zwp_idle_inhibit_manager_v1',                version:  1, name: 27
interface: 'zwp_relative_pointer_manager_v1',            version:  1, name: 28
interface: 'zwp_pointer_constraints_v1',                 version:  1, name: 29
interface: 'zwlr_input_inhibit_manager_v1',              version:  1, name: 30
interface: 'zwlr_foreign_toplevel_manager_v1',           version:  3, name: 31
interface: 'ext_session_lock_manager_v1',                version:  1, name: 32
interface: 'wp_drm_lease_device_v1',                     version:  1, name: 33
	path: /dev/dri/card1
	connector:
		id: 65
		name: DP-1
		description: Acer Technologies GF246 T8CEE0018521 (DP-1)
interface: 'zwlr_output_power_manager_v1',               version:  1, name: 34
interface: 'zwlr_layer_shell_v1',                        version:  4, name: 35
interface: 'wl_output',                                  version:  4, name: 36
	name: DP-1
	description: Acer Technologies GF246 T8CEE0018521 (DP-1)
	x: 0, y: 0, scale: 1,
	physical_width: 530 mm, physical_height: 300 mm,
	make: 'Acer Technologies', model: 'GF246',
	subpixel_orientation: unknown, output_transform: normal,
	mode:
		width: 1920 px, height: 1080 px, refresh: 60.000 Hz,
		flags: current
```

Looking at the hikari output it seems I misremembered something, as it doesn't support any of the idle protocols. Maybe I mistook it with the automatic timeout? I dunno.

**Comment by fwsmit** [comment]
Does it work on labwc? It has the `org_kde_kwin_idle` protocol, so it should work

**Comment by Narrat** [comment]
Doesn't work on labwc even with the protocol support present

**Comment by fwsmit** [comment]
Hmm, interesting. Does it work on sway? Do they still have the protocol? I have originally tested it on sway.

One thing you can do to test is set the idle threshold to some low value and run Dunst with debug printing. I believe Dunst should print when it things you are idle for more than the idle timeout

**Comment by Narrat** [comment]
Setting the option explicitly in the config results in this message: `WARNING: couldn't set idle timeout`
Dunno what the default value is set to (IIRC something like 10min?), but the status doesn't change even after 30min and more. It does get the active status (`INFO: Idle status queried: 0`), but that never changes.

Didn't test with sway yet.

**Comment by Narrat** [comment]
A new wlroots version is around (0.17.0) and it has this in the changelog:
> drop KDE idle protocol support ([!3778](https://github.com/wlroots/wlroots/-/merge_requests/3778)): this KDE protocol has been superseded by the standard idle-notify-v1 protocol.

**Comment by fwsmit** [comment]
Should be fixed with #1250
